### DISCUSSION TITLE
Odin (programming language) won't install on M1 CPU

### DISCUSSION THREAD

**Discussion by vascanera** [Homebrew/discussions#1654] [Everyday usage]
Odin won't install on M1 architecture. On the MAKE step, it breaks with an error "Unknown CPU Type". 
Compiling Odin from the cloned repo with "make" command works ok (the CPU architecture is recognized).
Please help updating the formulae for Odin. (https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/odin.rb)
Thanks!

**Comment by gromgit** [comment]
Not a maintainer, but I'm very sure that won't happen till upstream cuts a new stable release.

> Compiling Odin from the cloned repo with "make" command works ok (the CPU architecture is recognized).

Then you're building HEAD from the repo. You can do the same thing in Homebrew (since the formula has a `head` block):
```
brew install --HEAD odin
```
but you'll be building it from source yourself.
